Source/Core/DolphinQt2/GameList/GameListModel.cpp

```diff
@@ -138,6 +138,63 @@ int GameListModel::columnCount(const QModelIndex& parent) const
   return NUM_COLS;
 }
 
+bool GameListModel::ShouldDisplayGameListItem(const QModelIndex& index) {
```

Comment:
Since only the row number is being used, and the functions that call this specifically build up a `QModelIndex` just for this, can we pass the row number directly? That'd make the `filterAcceptsRow` functions a bit simpler.